test/source/mock/fes/fes-endpoints.ts

```diff
@@ -55,6 +55,21 @@ export const mockFesEndpoints: HandlersDefinition = {
     }
     throw new HttpClientErr('Not Found', 404);
   },
+  '/api/v1/message': async ({ }, req) => {
+    if (req.headers.host === standardFesUrl && req.method === 'POST') {
+      authenticate(req, 'fes');
+      return { 'url': `http://${standardFesUrl}/message/FES-MOCK-MESSAGE-ID` };
+    }
+    throw new HttpClientErr('Not Found', 404);
+  },
+  '/api/v1/message/new-reply-token': async ({ }) => { // why is this not getting called?
+    throw new HttpClientErr('why not getting called?', 400);
```

Comment:
To investigate - this API is expected to get called before `POST /api/v1/message` but it doesn't.

Reply:
It was because of missing `auth.uuid` meant for the flowcrypt.com/api.
 - when using FES, not relevant
 - when using flowcrypt.com/api, it's better to leave it missing and make request anyway. backend will produce auth error, which will cause extension to render a prompt and re-auth the user